src/main/java/teammates/common/datatransfer/StudentAttributes.java

```diff
@@ -130,7 +129,7 @@ public boolean isRegistered() {
     }
 
     public String getRegistrationUrl() {
-        return new Url(Config.APP_URL + Const.ActionURIs.STUDENT_COURSE_JOIN_NEW)
+        return new Url(Const.ActionURIs.STUDENT_COURSE_JOIN_NEW)
```

Comment:
The other URL method `getPublicProfilePictureUrl()` returns relative URL without any indication whatsoever. Come to think of it, it was very odd before: one method returned absolute URL and the other returned relative. Good that it's resolved now.
It should be either nothing is changed or both methods are renamed to `getRelative*Url()`.